xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.cpp

```diff
@@ -201,7 +202,8 @@ enum AVPixelFormat CDVDVideoCodecDRMPRIME::GetFormat(struct AVCodecContext* avct
     }
   }
 
-  CLog::Log(LOGERROR, "CDVDVideoCodecDRMPRIME::{} - unsupported pixel format", __FUNCTION__);
+  CLog::Log(LOGERROR, "CDVDVideoCodecDRMPRIME::{} - unsupported pixel format: {}", __FUNCTION__,
+            av_get_pix_fmt_name(fmt[0]));
```

Comment:
I don't think this is quite right as the fmt is an array of supported formats.

> is the list of formats which are supported by the codec, it is terminated by -1 as 0 is a valid format, the formats are ordered by quality. The first is always the native one.

ref: https://ffmpeg.org/doxygen/4.1/structAVCodecContext.html#ae85c5a0e81e9f97c063881148edc28b7

Reply:
I was assuming the native format (first entry) was the important part.
That will likely indicate if the codec settings are natively 12-bit or YUV444 which if the significant reason we can't handle it.

Would you prefer:
 Remove this logging change
 Change wording to: "unsupported native pixel format: {}"
 build up a string listing all formats

Reply:
Updated